datasets/shear_flow/README.md

```diff
@@ -57,7 +57,7 @@ Table: VRMSE metrics on test sets (lower is better). Best results are shown in b
 
 **Grid type:** uniform, cartesian coordinates.
 
-**Initial conditions:** the shear field $u_1x$ is composed of $n_\text{shear}$ shears uniformly spaced along the $z$ direction. Each shear is implemented with a tanh (hyperbolic tangent) $\text{tanh}(5\frac{y-y_k}{n_\text{shear}w})$ where $z_k$ is the vertical position of the shear and $w$ is a width factor.
+**Initial conditions:** the shear field $u_x$ is composed of $n_\text{shear}$ shears uniformly spaced along the $y$ direction. Each shear is implemented with a tanh (hyperbolic tangent) $\text{tanh}(5\frac{y-y_k}{n_\text{shear}w})$ where $z_k$ is the vertical position of the shear and $w$ is a width factor.
```

Comment:
This is tricky because the HDF5 file saves the axes as `x, y` but the data generation is `x, z`. It looks like other sections of the doc specify `y` so maybe the `z` should be `y` here with a note somewhere indicating `y` is `z` in the generating code?

Reply:
yes, I think the README should be coherent with the .hdf5 files (x,y)